src/views/radio.rs

```diff
@@ -70,6 +74,13 @@ impl<T> RadioGroup<T> {
     pub fn selection(&self) -> Rc<T> {
         self.state.borrow().selection()
     }
+
+    /// Sets a callback to be used when the selection changes.
+    pub fn set_on_change<F: 'static + Fn(&mut Cursive, &T)>(
```

Comment:
Can you add a chainable `pub fn on_change<F>(self, on_change: F) -> Self` as well?

Reply:
Done